settings.gradle

```diff
@@ -35,4 +35,5 @@ include 'data-prepper-plugins:otel-trace-group-prepper'
 include 'data-prepper-plugins:otel-trace-source'
 include 'data-prepper-plugins:peer-forwarder'
 include 'data-prepper-plugins:blocking-buffer'
+include 'data-prepper-plugins:grok-prepper'
```

Comment:
Why did you add this dependency? In general, our plugins should not depend on each other.

Reply:
The settings.gradle file defines all of the internal projects. All of the plugin projects are required to go in here.

Reply:
Yes, you are correct on this.